test/test_other.py

```diff
@@ -2569,28 +2569,9 @@ def test_emcc_debug_files(self):
           if debug is None:
             self.assertFalse(os.path.exists(self.canonical_temp_dir))
           elif debug == '1':
-            self.assertExists(os.path.join(self.canonical_temp_dir, 'emcc-3-original.js'))
+            self.assertExists(os.path.join(self.canonical_temp_dir, 'emcc-4-original.js'))
```

Comment:
This is because the order of stripping changed:

Before:
```console
emcc-0-base.wasm 
emcc-1-wasm-emscripten-finalize.wasm
emcc-2-post_finalize.wasm
emcc-3-original.js
emcc-4-strip.wasm
...
```
After:
```console
emcc-0-base.wasm
emcc-1-strip.wasm
emcc-2-wasm-emscripten-finalize.wasm
emcc-3-post_finalize.wasm
emcc-4-original.js
...
```